app/src/main/java/com/grindrplus/Hooker.kt

```diff
@@ -50,7 +55,6 @@ class Hooker : IXposedHookLoadPackage {
 
                     if (pkgVersionName != TARGET_PKG_VERSION_NAME) {
                         toastInvalidVersionName()
-                        return
```

Comment:
Allow hook newer or older version

Reply:
Cannot allow the user to do that. There's a good reason for this: symbol obfuscation mapping varies per version, so if you try to run the hooks with the wrong version, expect undefined behavior and weird bugs.

Reply:
Oh I didn't thought about that before sorry